src/vmware/azext_vmware/_help.py

```diff
@@ -712,3 +712,172 @@
+    type: command
+    short-summary: Get a segment by ID in a private cloud workload network.
+    examples:
+    - name: Get a segment by ID in a workload network.
+      text: az vmware workload-network segment show --resource-group group1 --private-cloud cloud1 --segment-id segment1
+"""
+
+helps['vmware workload-network segment create'] = """
+    type: command
+    short-summary: Create a segment by ID in a private cloud workload network.
+    examples:
+    - name: Create a segment by ID in a workload network.
+      text: az vmware workload-network segment create --resource-group group1 --private-cloud cloud1 --segment-id segment1 --display-name segment1 --connected-gateway /infra/tier-1s/gateway --revision 1 --dhcp-ranges 40.20.0.0 40.20.0.1 --gateway-address 40.20.20.20/16
+"""
+
+helps['vmware workload-network segment update'] = """
+    type: command
+    short-summary: Update a segment by ID in a private cloud workload network.
+    examples:
+    - name: Update a segment by ID in a workload network.
+      text: az vmware workload-network segment update --resource-group group1 --private-cloud cloud1 --segment-id segment1 --display-name segment1 --connected-gateway /infra/tier-1s/gateway --revision 1 --dhcp-ranges 40.20.0.0 40.20.0.1 --gateway-address 40.20.20.20/16
+"""
+
+helps['vmware workload-network segment delete'] = """
+    type: command
+    short-summary: Delete a segment by ID in a private cloud workload network.
+    examples:
+    - name: Delete a segment by ID in a workload network.
+      text: az vmware workload-network segment delete --resource-group group1 --private-cloud cloud1 --segment-id segment1
+"""
+
+helps['vmware workload-network public-ip'] = """
+    type: group
+    short-summary: Commands to manage a Public-IP workload network.
+"""
+
+helps['vmware workload-network public-ip list'] = """
+    type: command
+    short-summary: List of Public IP Blocks in a private cloud workload network.
+    examples:
+    - name: List of Public IP Blocks in a workload network.
+      text: az vmware workload-network public-ip list --resource-group group1 --private-cloud cloud1
+"""
+
+helps['vmware workload-network public-ip show'] = """
+    type: command
+    short-summary: Get a Public IP Block by ID in a private cloud workload network.
+    examples:
+    - name: Get a Public IP Block by ID in a workload network.
+      text: az vmware workload-network public-ip show --resource-group group1 --private-cloud cloud1 --public-ip-id publicIP1
+"""
+
+helps['vmware workload-network public-ip create'] = """
+    type: command
+    short-summary: Create a Public IP Block by ID in a private cloud workload network.
+    examples:
+    - name: Create a Public IP Block by ID in a workload network.
+      text: az vmware workload-network public-ip create --resource-group group1 --private-cloud cloud1 --public-ip-id publicIP1 --display-name publicIP1 --number-of-public-i-ps 32

```

Comment:
The spec is `numberOfPublicIPs`.  Please make the command line arg `--number-of-public-ips`. The generated Python code is unfortunate.

Reply:
Updated